hardware_layer/hardware_imu/src/yawDirect.cpp

```diff
@@ -35,8 +35,7 @@ int main(int argc, char **argv)
     read_sensors();
     msg.data = TO_DEG(-atan2(magnetom[1], magnetom[0]));
     chatter_pub.publish(msg);
-    ROS_INFO("%s %f", "send an imu message", TO_DEG(-atan2(magnetom[1], magnetom[0])));
-    // ROS_INFO("%d \n",temp);
+    ROS_INFO("%s %f", "yawDirect: send an imu message", TO_DEG(-atan2(magnetom[1], magnetom[0])));
```

Comment:
you did `ros::this_node::getName().c_str()`  above and have hardcoded `yawDirect` here ? this is unacceptable @soumye . @pkhrag please read the changes before accepting diffs. You guys will be reviewing pull requests from y16 soon, learn to do code review properly.